audio_context.html

```diff
@@ -0,0 +1,321 @@
+<!doctype html>
+<head>
+  <title>Web Audio Api - Audio Context</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+
+<style>
+
+  :root {
+    --flex-direction: column;
+    --line-height: 4em;
+  }
+
+  @media(min-width: 250px) {
+    :root {
+      --flex-direction: row;
+      --line-height: 8em;
+    }
+  }
+
+  h1 {
+    text-align: center;
+  }
+
+  main {
+    outline: thick double;
+    background-color: whitesmoke;
+  }
+
+  fieldset {
+    display: flex;
+    border: none;
+    background: yellow;
+  }
+
+  form > fieldset {
+    flex-direction: var(--flex-direction);
+  }
+
+  fieldset fieldset {
+    background: red;
+  }
+
+  fieldset input[type="range"] {
+    width: 20vw;
+    display: inline-block;
+    top: 0.3em;
+    position: relative;
+  }
+
+  form > fieldset > label {
+    background: turquoise;
+    line-height: var(--line-height);
+  }
+
+  form > header {
+    background: yellow;
+  }
+
+  [type="checkbox"] {
+  }
+
+</style>
+
+</head>
+
+<body>
+
+  <header>
+    <h1>DevPunks Sequencer</h1>
```

Comment:
Headers on headers? Seems like duplicated content w form header and masthead.  🤔

Reply:
One of the headers was for the masthead of the page and the other was for the sequencer, I could change it to another element like an H2 but I remember us talking about having a header & two nested <label> tags with nested input tags inside of them.I could also change the name the masthead to something else ?

Reply:
see below...